README.md

```diff
@@ -107,7 +107,7 @@ Once you’ve done that, restart the server so it uses the new configuration inf
 
 Open up a browser and visit the URL for your application server's **Access Token endpoint**: `https://{YOUR_SERVER_URL}/accessToken` (If your app server is written in PHP, then you need `.php` extension at the end). If everything is configured correctly, you should see a long string of letters and numbers, which is a Twilio Access Token. Your iOS app will use a token like this to connect to Twilio.
 
-### <a name="bullet6"></a>6. Run the app
+### <a name="bullet6"></a>6. Run the Swift Quickstart app

```

Comment:
In the "table of contents above" `bullet6` is defined as `Run the app`. Maybe make that change above as well?